codalab/worker/bundle_manager.py

```diff
@@ -529,3 +533,20 @@ def _construct_run_message(self, worker, bundle):
         resources['request_network'] = bundle.metadata.request_network
 
         return message
+
+    def _bring_zombie_bundles_to_fail(self):
+        failure_message = "Bundle exceeds maximum time limit {}.".format(BUNDLE_TIMEOUT_SECONDS)
```

Comment:
This is confusing since the bundle time limit is another thing (`--request-time`).  Instead: Bundle has not responded in {} days.  I'd make it clear what the units here are.